neon_utils/signal_utils.py

```diff
@@ -35,6 +35,43 @@
 _BUS: MessageBusClient = None
 _MAX_TIMEOUT: int = None
 
+_create_signal = None
+_check_for_signal = None
+_wait_for_signal_clear = None
+_wait_for_signal_create = None
+
+
```

Comment:
would it work to push all of the signal related methods to some class, to reduce import diversity?

Reply:
Main issue is backwards-compatibility; we're basically overriding two methods (`create_signal` and `check_for_signal`).

That is a good idea though if we want to maintain some signal implementation going forward